sympy/polys/partfrac.py

```diff
@@ -52,7 +52,7 @@ def apart(f, x=None, full=False, **options):
     You can choose Bronstein's algorithm by setting ``full=True``:
 
     >>> apart(y/(x**2 + x + 1), x, full=True)
-    RootSum(_w**2 + _w + 1, Lambda(_a, (-2*_a*y/3 - y/3)/(-_a + x)))
+    RootSum(_x**2 + _x + 1, Lambda(_x, (-2*_x*y/3 - y/3)/(-_x + x)))
```

Comment:
There are a lot of cases like this where other symbols are also called `x`. It might be better to have the internal Dummy symbol be called something more distinct like `_1` or perhaps `w` is better than `x`.

Reply:
Didn't `lambda` get used somewhere at sometime for this symbol/

Reply:
I think `lambda` is awkward because it is a keyword in Python:
```python
In [5]: l = Symbol('lambda')

In [6]: print(l**2 + l + 1)
lambda**2 + lambda + 1

In [7]: lambda**2 + lambda + 1
  Cell In[7], line 1
    lambda**2 + lambda + 1
            ^
SyntaxError: invalid syntax
```
Also RootOf expressions can be long so you really want a one or at most two letter symbol.

Reply:
Agree -- I thought it printed in latex, however. But if the string prints the full name then a single letter is better. The printer could be modified to use lambda. What about just "_"? `RootSum(_**2 + _ + 1, Lambda(_, (-2*_*y/3 - y/3)/(-_ + x)))`?

Reply:
I think that `_` would be confusing. It would just be better to use a symbol that is not `x` (or `_x`) because that will often clash. Probably `w` as is used by `RootSum` on master is better.